tools/eksDistroBuildToolingOpsTools/cmd/eksGoTool/cmd/backportIssue.go

```diff
@@ -0,0 +1,160 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	gogithub "github.com/google/go-github/v48/github"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/constants"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/github"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/issueManager"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/logger"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/repoManager"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/retrier"
+)
+
+const (
+	// Flag Names
+	toplevelIssueIdFlag = "toplevelIssueId"
+	backportVersionFlag = "backportVersions"
+
+	// Other constants
+	issueAutocreatedTemplate   = "Issue was generated by eksGoTools createCveIssue, by:"
```

Comment:
mind if we put these in `pkg/constants/constants.go`? We can prolly de-dupe eksGoOwner and eksGoRepo too